public-env-vars.toml

```diff
@@ -0,0 +1,110 @@
+# TOML file for the `/env` route of Config endpoint
+# This file is used when the Config module is enabled.
+# It specifies a list of environment variable names (`variables`) 
+# that will be exposed by the `/env` route.
+
+
+variables = [
+    "ESPRESSO_SEQUENCER_ORCHESTRATOR_URL",
+    "ESPRESSO_SEQUENCER_CDN_ENDPOINT",
+    "ESPRESSO_SEQUENCER_LIBP2P_BIND_ADDRESS",
+    "ESPRESSO_SEQUENCER_LIBP2P_ADVERTISE_ADDRESS",
+    "ESPRESSO_STATE_RELAY_SERVER_URL",
+    "ESPRESSO_SEQUENCER_PREFUNDED_BUILDER_ACCOUNTS",
+    "ESPRESSO_SEQUENCER_L1_PROVIDER",
+    "ESPRESSO_SEQUENCER_L1_GENESIS",
+    "ESPRESSO_SEQUENCER_L1_EVENTS_MAX_BLOCK_RANGE",
+    "ESPRESSO_SEQUENCER_IS_DA",
+    "ESPRESSO_SEQUENCER_STATE_PEERS",
+    "ESPRESSO_SEQUENCER_STAKE_TABLE_CAPACITY",
+    "ESPRESSO_SEQUENCER_API_PEERS",
+    "ESPRESSO_SEQUENCER_HOTSHOT_EVENT_STREAMING_API_PORT",
+    "ESPRESSO_SEQUENCER_POSTGRES_HOST",
+    "ESPRESSO_SEQUENCER_POSTGRES_PORT",
+    "ESPRESSO_SEQUENCER_POSTGRES_DATABASE",
+    "ESPRESSO_SEQUENCER_POSTGRES_USER",
+    "ESPRESSO_SEQUENCER_POSTGRES_USE_TLS",
+    "ESPRESSO_SEQUENCER_POSTGRES_PRUNE",
+    "ESPRESSO_SEQUENCER_PRUNER_PRUNING_THRESHOLD",
+    "ESPRESSO_SEQUENCER_PRUNER_MINIMUM_RETENTION",
+    "ESPRESSO_SEQUENCER_PRUNER_TARGET_RETENTION",
+    "ESPRESSO_SEQUENCER_PRUNER_BATCH_SIZE",
+    "ESPRESSO_SEQUENCER_PRUNER_MAX_USAGE",
+    "ESPRESSO_SEQUENCER_PRUNER_INTERVAL",
+    "ESPRESSO_SEQUENCER_STORE_UNDECIDED_STATE",
+    "ESPRESSO_SEQUENCER_API_PORT",
+    "ESPRESSO_SEQUENCER_STORAGE_PATH",
+    "ESPRESSO_SEQUENCER_CHAIN_ID",
+    "ESPRESSO_SEQUENCER_MAX_BLOCK_SIZE",
+    "ESPRESSO_SEQUENCER_BASE_FEE",
+    "ESPRESSO_SEQUENCER_FEE_CONTRACT_PROXY_ADDRESS",
+    "ESPRESSO_SEQUENCER_FEE_RECIPIENT",
+    "ESPRESSO_CDN_BROKER_DISCOVERY_ENDPOINT",
+    "ESPRESSO_CDN_BROKER_PUBLIC_BIND_ENDPOINT",
+    "ESPRESSO_CDN_BROKER_PUBLIC_ADVERTISE_ENDPOINT",
+    "ESPRESSO_CDN_BROKER_METRICS_BIND_ENDPOINT",
+    "ESPRESSO_CDN_BROKER_CA_CERT_PATH",
+    "ESPRESSO_CDN_MARSHAL_DISCOVERY_ENDPOINT",
+    "ESPRESSO_CDN_MARSHAL_BIND_PORT",
+    "ESPRESSO_CDN_MARSHAL_METRICS_BIND_ENDPOINT",
+    "ESPRESSO_CDN_MARSHAL_CA_CERT_PATH",
+    "ESPRESSO_CDN_WHITELIST_DISCOVERY_ENDPOINT",
+    "ESPRESSO_SEQUENCER_URL",
+    "ESPRESSO_SEQUENCER_HOTSHOT_ADDRESS",
+    "ESPRESSO_SEQUENCER_HOTSHOT_ACCOUNT_INDEX",
```

Comment:
I don't think we should expose these. Technically it's useless without the mnemonic, but it is one of the inputs that feeds into the private key derivation so feels a little too close to a secret

Reply:
Sorry, I meant to hide any account index variables. I think you deleted the 4 lines shown here but I only meant to comment on 54 (github just shows the rest for context)

Reply:
np I'll add those back. Is the ACCOUNT_INDEX different from this one?